recipes/macsyfinder/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "macsyfinder" %}
+{% set version = "2.0rc1" %}
```

Comment:
Do you really want to distribute a release candidate rather than a final release?

Reply:
Hi, thanks for check this. I am trying to build another tool (CRISPRCasFinder) that depends on MacSyFinder, I will update this when the release is completed